README Overhaul (#1011)
wabbajack-tools/wabbajack: README.md

```ini
[General]
directURL=http://skse.silverlock.org/beta/skse64_2_00_18.7z
```

In the example above we used [SKSE SE 2.0.18](http://skse.silverlock.org/) and you'd have to replace the URL with the direct download link of the Script Extender you are using.

Next up are Cleaned Master Files. Use xEdit through MO2 and clean the base game master files normally, copy the cleaned master files to a new mod in MO2 (called `Cleaned Master Files` or something) and replace the cleaned master files in your game folder with the backup created by xEdit to revert them to their default unmodified state.

#### Game Folder Files and MO2

You now know about `.meta` files and understand that every file has to come from somewhere, this also means that you also need to create `.meta` files for MO2 and files in your game folder. We recommend using the [MO2 GitHub Releases](https://github.com/ModOrganizer2/modorganizer/releases/) instead of the ones you find on the Nexus.

> Wabbajack will not only index your entire MO2 folder but also your entire. Mods that need to be installed to the game folder by the user are required to be installed to a `Game Folder Files` folder in your MO2 directory. Let's look at an example for better understanding:

Reviewer: ```suggestion
Mods that need to be installed to the game folder by the user are required to be installed to a `Game Folder Files` folder in your MO2 directory. Let's look at an example for better understanding:
```

> Wabbajack will not only index your entire MO2 folder but also your entire. Mods that need to be installed to the game folder by the user are required to be installed to a `Game Folder Files` folder in your MO2 directory. Let's look at an example for better understanding:

Reviewer: ```suggestion
Mods that need to be installed to the game folder by the user are required to be installed to a `Game Folder Files` folder in your MO2 directory. Let's look at an example for better understanding:
```

> Wabbajack will not only index your entire MO2 folder but also your entire Game folder and will try to also match files from your downloads folder with your game folder. These matched files will end up in a `Game Folder files` folder in the MO2 subdirectory after a user installs your list. Let's look at an example for better understanding:

Reviewer: Didn't this change, that Wabbajack now indexes the entire game folder, and uses a matching game folder to store information about files that are in a MO2 mod but not in the downloads folder, to store the information to recreate this mods files with a binary patch (if there is a file missing in the downloads), and that authors have to create a `Game Folder Files` folder by themself.

Reviewer: Guess I was right with the game folder files halgari just :literallythis: reacted to my post about this on Discord.

> Wabbajack will not only index your entire MO2 folder but also your entire Game folder and will try to also match files from your downloads folder with your game folder. These matched files will end up in a `Game Folder files` folder in the MO2 subdirectory after a user installs your list. Let's look at an example for better understanding:

Reviewer: ```suggestion
Wabbajack will not only index your entire MO2 folder but also your entire game folder and will try to also match files from your game folder to your MO2 folder, if it can't find a source in your downloads and if they are changed save those changes. For mods that need to be installed in the game folder by the user you will have to create a 'Game Folder Files' folder in your MO2 main directory, which is where the `ModOrganizer.exe` is located. Let's look at an example for better understanding:
```

> You will likely make use of SKSE/F4SE/OBSE or other Script Extender if you are modding Bethesda titles. Eg for SKSE you'd need to install `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` to your game folder. To make things easier for the end user, you can also put those files inside the `Game Folder Files` folder at `MO2\Game Folder Files`. You also need the archive and it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't) in your downloads folder so Wabbajack can match those files.

Reviewer: ```suggestion
You will likely make use of SKSE/F4SE/OBSE or other Script Extender if you are modding Bethesda titles. Eg for SKSE you'd need to install `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` to your game folder. To make things easier for the end user, you should also put those files inside the `Game Folder Files` folder at `MO2\Game Folder Files`. You also need the archive and it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't) in your downloads folder so Wabbajack can match those files.
```

Reviewer: "can"->"should"

> You will likely make use of SKSE/F4SE/OBSE or other Script Extender if you are modding Bethesda titles. Eg for SKSE you'd need to install `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` to your game folder. To make things easier for the end user, you can also put those files inside the `Game Folder Files` folder at `MO2\Game Folder Files`. You also need the archive and it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't) in your downloads folder so Wabbajack can match those files.

Reviewer: ```suggestion
You will likely make use of SKSE/F4SE/OBSE or other Script Extender if you are modding Bethesda titles. Eg for SKSE you'd need to install `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` to your game folder. To make things easier for the end user, you should also put those files inside the `Game Folder Files` folder at `MO2\Game Folder Files`. You also need the archive and it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't) in your downloads folder so Wabbajack can match those files.
```

> In your game folder you will likely have SKSE/F4SE/OBSE and in your downloads folder you would have the SKSE archive with it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't). Wabbajack will find the script extender files in your game folder, match them against the archive in your downloads folder and will put eg `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` in the `Game Folder Files` folder during installation.

Reviewer: ```suggestion
The prime example of files that are matched from your game folder are the cleaned master files (.esm) of your vanilla Bethesda game which wont come from any mod (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't). You likely also want to use SKSE/F4SE/OBSE and in your downloads folder you would have the SKSE archive with it's `.meta` file (see [Script Extender and Cleaned ESMs](#script-extender-and-cleaned-masters) section if you haven't). But those need to be installed in the game folder of the user, which Wabbajack won't do by itself, which is why you need to put for example `skse64_1_5_97.dll`, `skse64_loader.exe` and `skse64_steam_loader.dll` in the `Game Folder Files` folder you created inside your MO2 directory, which will then be recreated on the users machine upon installation.
```

> The user only has to copy the files in the `Game Folder Fils` directory to their game folder after installation in order for them to install the same files you have. This also works with other files in your game folder as long as you have a download with those files and a `.meta` file for them, eg: ENBs.

Reviewer: ```suggestion
The user only has to copy the files in the `Game Folder Files` directory to their game folder after installation in order for them to install the same files you have. This also works with other files in your game folder as long as you have a download with those files and a `.meta` file for them, eg: ENBs.
```

> 
> The user only has to copy the files in the `Game Folder Fils` directory to their game folder after installation in order for them to install the same files you have. This also works with other files in your game folder as long as you have a download with those files and a `.meta` file for them, eg: ENBs.

Reviewer: ```suggestion

The user then only has to copy the files in the `Game Folder Fils` directory to their game folder after installation in order for them to install the same files you have. This has to be done with every mod that needs to be installed in your game folder and you have a download with those files and a `.meta` file for them, eg: ENBs.
```

In the Wabbajack community we differentiate between _official_ and _unofficial_ Modlists. The former being curated Modlists which have been tested and are available in the Wabbajack Gallery while the latter are Modlists submitted on our [Discord](https://discord.gg/wabbajack) in the `unofficial-modlist-submissions` channel and are not tested.

Official Modlists also get their own support channel on our Discord and often have 500-2500 users depending on the game. Do note that these Modlists receive regular updates and the authors are putting in a lot of hours to deliver the best possible version. If you think your Modlist and more importantly: yourself are up for the task you can submit your Modlist for official status on the Discord in the `modlist-development-info` channel.

Managing a Modlist can be tricky if you have never done something like this. In the early days of Wabbajack we mostly relied on GDocs READMEs and hosted all accompanying files on GDrive while managing issues through the support channel on Discord. This system was later almost completely replaced through valiant effort by one of our developers: almost all new Modlists use GitHub for managing their Modlist.

> GitHub was made for developers and is the site you are currently on. It mostly hosts source code for open source projects, such as Wabbajack, but can also be used for project management. Another strong point is Markdown support. This README you are currently reading is, like very other GitHub README, written in Markdown and rendered on GitHub.

Reviewer: ```suggestion
GitHub was made for developers and is the site you are currently on. It mostly hosts source code for open source projects, such as Wabbajack, but can also be used for project management. Another strong point is Markdown support. This README you are currently reading is, like every other GitHub README, written in Markdown and rendered on GitHub.
```
